cube/src/main/java/zone/cogni/asquare/cube/operation/OperationResultProcessor.java

```diff
@@ -176,18 +176,17 @@ private SingleGroupResult findValidateGroup(SingleGroupResult root, String curre
                                           String operationGroupId) {
     Context context = Context.create(permissionsSupplier, modelSupplier);
 
-    OperationGroup operationGroup = operationRoot.getByPathId(operationGroupId);
-
-    List<SingleGroupResult> result = uris.stream()
-                                         .map(uri -> {
-                                           SingleGroupResult rootGroup = new SingleGroupResult(operationGroup, uri, null);
-                                           processGroup(context, rootGroup);
-                                           return rootGroup;
-                                         })
-                                         .collect(Collectors.toList());
-
-    log.debug("(validate) {}", context.monitor);
-    return result;
+    try {
```

Comment:
yes this was probably never adapted for the multi-file solution, but in stead of duplicating code, here it probably makes sense have the method below this one call this method by delegation. also move the method below up (since it will be calling this one)